public/javascripts/tool_inline.js

```diff
@@ -137,61 +138,4 @@ $(function() {
   })
 });
 
-window.addEventListener('message', function(e) {
-  try {
-    var message = JSON.parse(e.data);
-    switch (message.subject) {
-      case 'lti.frameResize':
-        const toolResizer = new ToolLaunchResizer();
-        var height = message.height;
-        if (height <= 0) height = 1;
-
-        const container = toolResizer.tool_content_wrapper(message.token || e.origin).data('height_overridden', true);
```

Comment:
It would be good to continue to use the toolResizer, if you can get the e.source to work with resizing the container containing the iframe, then I would update the toolResizer to use e.source instead

Reply:
I've changed the code back to use the toolResizer and added tests.